src/Compilers/Core/Portable/Symbols/CustomModifier.cs

```diff
@@ -18,20 +18,6 @@ public abstract class CustomModifier : Cci.ICustomModifier
         /// </summary>
         public abstract INamedTypeSymbol Modifier { get; }
```

Comment:
>public abstract INamedTypeSymbol Modifier { get; } [](start = 8, length = 50)

What can we do to ensure that this member is not used when it is not supposed to?

Reply:
You could always call ICustomModifier.GetModifier. Is there a scenario you're concerned about where it was safe to call that method, but not this one?

Reply:
My concern is that it requires special knowledge when it is appropriate to use this API. Before this change no one could make this mistake because of the interface, now (based on my experience) this is going to happen all the time.